lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,26 @@
+module Semian
+  class AtomicInteger < SharedMemoryObject #:nodoc:
+
+    def initialize(name, permissions)
+      data_layout = [:int]
+      if acquire(name, data_layout, permissions)
+      else
+        @value=0
+      end
```

Comment:
Oh, the idea is that if it can call `acquire` (and it does something), it's implemented in C?